src/helpDB.jl

```diff
@@ -0,0 +1,296 @@
+# This file is part of the TaylorSeries.jl Julia package, MIT license
+#
+#`docstrings` for the Taylor1.jl and TaylorN.jl scripts
+#
+
+
+                                        ### Exclusive Operations ###
+
+
+
+## Constructors ##
+@doc """
+    Taylor1{T<:Number} <: Number
+
+DataType for polynomial expansions in one independent variable.
+
+**Fields:**
+
+- `coeffs :: Array{T,1}` Expansion coefficients; the \$i\$-th
+component is the coefficient of degree \$i-1\$ of the expansion.
+- `order  :: Int64` Maximum order (degree) of the polynomial.
+"""->
+Taylor1
+
+@doc """
+    taylor1_variable(T, [order=1])
+    taylor1_variable([order=1])
+
+Short-cut to define the independent variable as a `Taylor1` polynomial of
+given `order`. If `T::Type` is ommitted, `Float64` is assumend.
+"""-> 
+taylor1_variable
+
+## get_coeff ##
+@doc """
+    get_coeff(a, n)
+
+Return the coefficient of order `n::Int` of a `a::Taylor1` polynomial.
+""" ->
+get_coeff
+
+## Differentiating ##
+doc"""
+    diffTaylor(a)
+
+Return the `Taylor1` polynomial of the differential of `a::Taylor1`. 
+
+The last coefficient is set to zero.
+"""
+diffTaylor
+
+## Integrating ##
+doc"""
+    integTaylor(a, x)
+    integTaylor(a)
+
+Returns the integral of `a::Taylor1`. 
+
+The constant of integration (0th order coefficient) is set to `x`, which is zero if ommitted.
+"""
+integTaylor
+
+## Evaluating ##
+doc"""
+    evaluate(a, dx)
+    evaluate(a)
+
+Evaluates a `Taylor1` polynomial using Horner's rule (hand coded).
+
+
+    evaluate(a, x)
+
+Returns the substitution of `x::Taylor1` as independent variable in
+`a::Taylor1`.
+"""
+evaluate
+
+## Returns de n-th derivative of a series expansion
+doc"""
+    deriv(a, [n=1])
+
+Returns the value of the `n`-th derivative of `a`.
+"""
+deriv
+
+
+                                        ### Function Extensions ###
+
+
+## Exp ##
+doc"""
+    exp(a)
```

Comment:
At the beginning when I was playing with this, I was fed up with recompiling the package constantly, so I commented the relevant line in `TaylosSeries.jl`; if you uncomment it, it will recompile it and everything should work.